### ISSUE TITLE
Error: expected str, bytes or os.PathLike object, not NoneType

### ISSUE THREAD

**Issue by XenorInspire** [DGAC/inseca#1]
Hi !
I installed these dependencies : 
```bash
apt install python3
apt install python3-pacparser
apt install libgtk-3-dev
apt install git dbus
```
I downloaded rclone here : https://rclone.org/downloads/
I downloaded veracrypt here : https://www.veracrypt.fr/en/Downloads.html
Docker is installed and I did make commands.
`inseca -h` is working too.
Requests and sqlite3 modules seems to be embbeded with python3 (according internet researches)
But i have this error and I don't know where it comes from :
<br>
![image](https://user-images.githubusercontent.com/44984461/155106547-b7ab6a0b-4473-4979-aaa7-6411dee11af6.png)

**Comment by VivienMla** [comment]
Hi!
to see where is comes from, could you:
- send the result of: env | grep INSECA
- uncomment line 2232 of the tools/inseca file and re-run the config-create command. Hopefully you'll get a full exception stack which you can send here.

Regards,
Vivien

**Comment by XenorInspire** [comment]
Hi again!
Thanks for your answers.
First, the result of `env | greo INSECA` : 
![image](https://user-images.githubusercontent.com/44984461/155679289-cc6c039f-bfc4-485f-8d33-f4b7ecf8d235.png)

As you said, I uncommented the line 2232 and I've got this result : 
![image](https://user-images.githubusercontent.com/44984461/155679448-a20bbb2f-70f7-4243-8080-d9a718651cce.png)

Thank you for your time.
Regards.
